### ISSUE TITLE
Notification of "your branch is behind default branch" is not announced by screenreader

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6582]
The notification disrupts the tab order that was there, adding elements, could be confusing for screenreader users. When notification isn't shown there is a blue dot that is not exposed to screenreader users either.

**Comment by tidy-dev** [comment]
I cannot find a notification of "your branch is behind default branch" in the code base. Not sure what this is referring to so going to close.
